osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerDisc.cs

```diff
@@ -82,10 +82,12 @@ protected override bool OnMouseMove(MouseMoveEvent e)
 
         private float lastAngle;
         private float currentRotation;
-        public float RotationAbsolute;
+        public float BidirectionalRotation;
```

Comment:
i don't understand this variable name change. the old one makes more sense to me still?

Reply:
You wouldn't expect to find a variable named `RotationAbsolute` to be a negative value, which it may become when rewinding.

Reply:
It should *never* be negative.

Reply:
Ah correct, I was thinking differently for this one.